man/fi_rxm.7.md

```diff
@@ -44,13 +44,13 @@ result in a stall. See also the ERRORS section in fi_msg(3).
 
 # SUPPORTED FEATURES
 
-The RxM provider currently supports *FI_MSG*, *FI_TAGGED* and *FI_RMA* capabilities.
+The RxM provider currently supports *FI_MSG*, *FI_TAGGED*, *FI_RMA* and *FI_ATOMIC* capabilities.
```

Comment:
There is a caveat that should be mentioned here or in the verbs page. The FI_ATOMIC capability will not show up in the fi_info structure unless explicitly requested. A user who uses the fi_info utility wouldn't see it by default unless they provided that capability in the command line.

Reply:
Good point. I will update to add mention of this as part of the fi_rxm(7) man page.

Reply:
@jswaro - let me know if you think the updated text is sufficient. I also added the message order limitations if FI_ATOMIC is requested.